rfc/parameterizedMetrics.md

```diff
@@ -0,0 +1,124 @@
+Parameterized Metric Support
+============================
+
+Parameterized metrics are a subset of dynamic request metrics.  Specifically they focus on modifying the calculation of metrics by altering values used in the calculation of existing metrics or mapping their values at aggregation time.
+
+The expected contract of parameterized metrics is as follows: Given a base metric with api name `metricName`, it can be suffixed with a procedure-call list of keyword parameter names and values.  These values will be used in the calculation of the resulting new metric.
+
```

Comment:
When we talk about `new metric`, are we constraining result rows with parameters or transforming to a different metric? If it's the latter, do we need to configure metric dictionary or metric maker differently?

Reply:
We're building a new Logical Metric instance using a transformed TDQ and result set mapper (and name)

Reply:
Although I'm trying to restrict this document to API concerns for the most part.